spec/factories/trainees.rb

```diff
@@ -670,7 +669,14 @@
     end
 
     trait :with_hesa_trainee_detail do
+      with_hesa_student
```

Comment:
Will a Trainee with HESA trainee details (i.e. created via the API) have a HESA student record too?

Reply:
Removed `hesa_student` retained `hesa_id`